crates/next-core/src/next_config.rs

```diff
@@ -835,7 +836,6 @@
     adjust_font_fallbacks_with_size_adjust: Option<bool>,
     after: Option<bool>,
     app_document_preloading: Option<bool>,
-    cache_life: Option<FxIndexMap<String, CacheLifeProfile>>,
```

Comment:
An example of why this was a bad idea: This config was moved out of experimental, but Turbopack's config was never updated, so `edge-app-route.ts`​ was probably reading the wrong value when using Turbopack...